woocommerce/woocommerce-ios
# Show pink dot icon on Deprecated Stats Banner

closes #2465 

# Why

In order to make the deprecated stats banner stand out more, it should include a pink button on top of the sync image.

# How

I tried to draw the pink dot by code but it turns out to be quite complex when dealing with dark mode and RTL orientations. 
We have decided that a composed image would be the best for this case.

# Screenshots

Light | Dark | RTL
---- | ---- | ----
<img width="424" alt="light" src="https://user-images.githubusercontent.com/562080/86157692-64c64980-bacd-11ea-9eba-ae98715df243.png">|<img width="430" alt="dark" src="https://user-images.githubusercontent.com/562080/86157690-642db300-bacd-11ea-8458-a70824bfdadf.png">|<img width="413" alt="rtl" src="https://user-images.githubusercontent.com/562080/86157680-61cb5900-bacd-11ea-9aa3-db68d2644e60.png">


# Testing Steps

- Test the app with a store whose WooCommerce is lower than 4.0 or has the WCAdmin plugin disabled and see the stats banner with thew new sync image.


Update release notes:
